docs/topics/Community_Resources.md

```diff
@@ -23,10 +23,13 @@ The Discord team curates the following list of officially vetted libraries that
 | [Discordia](https://github.com/SinisterRectus/Discordia) | Lua |
 | [discord.py](https://github.com/Rapptz/discord.py) | Python |
 | [disco](https://github.com/b1naryth1ef/disco) | Python |
+| [RestCord](https://www.restcord.com/) | PHP* |
 | [discordrb](https://github.com/meew0/discordrb) | Ruby |
 | [discord-rs](https://github.com/SpaceManiac/discord-rs) | Rust |
 | [Sword](https://github.com/Azoy/Sword) | Swift |
 
+* REST Only Implementation
```

Comment:
We don't need this, all the libs here have various levels of implementations, and people can investigate on their own.

Reply:
sounds good